packages/opentelemetry-sdk-trace-base/src/MultiSpanProcessor.ts

```diff
@@ -47,7 +47,7 @@ export class MultiSpanProcessor implements SpanProcessor {
     });
   }
 
-  onStart(span: Span, context: Context): void {
+  onStart(span: WriteableSpan, context: Context): void {
```

Comment:
After some thinking, maybe we can just type it with `api.Span & ReadableSpan`?
These are also the interfaces that the current `SDK` is using:
```js
/**
 * This class represents a span.
 */
export class Span implements api.Span, ReadableSpan {
```